src/styles/images/index.md.njk

```diff
@@ -8,53 +8,70 @@ layout: layout-pane.njk
 
 {% from "_example.njk" import example %}
 
-Avoid unnecessary decoration. Only use images if there’s a real user need.
+Avoid using images for unnecessary decoration. Only use images if there’s a real user need.
```

Comment:
```suggestion
Avoid using images for decoration. Only use images if there’s a real user need.
```

Reply:
I think this qualifier's important, as decoration (such as branding) might be essential.

Reply:
Ah gotcha, yeah that makes sense.